src/index.ts

```diff
@@ -1,3 +0,0 @@
-import ts from './typescript'
```

Comment:
Why? Not all bundlers awaits typescript as module name.

In rollup-plugin-typescript2:

```ts
import patchedTypescript from 'ttypescript'
return {
  plugins: [
    tsPlugin({typescript: patchedTypescript})
  ]
```

Reply:
I'll fix main to `lib/typescript.js`